gitlab/v4/objects/files.py

```diff
@@ -112,11 +112,36 @@ class ProjectFileManager(GetMixin, CreateMixin, UpdateMixin, DeleteMixin, RESTMa
     @cli.register_custom_action(
         cls_names="ProjectFileManager", required=("file_path", "ref")
     )
-    # NOTE(jlvillal): Signature doesn't match UpdateMixin.update() so ignore
-    # type error
-    def get(  # type: ignore
-        self, file_path: str, ref: str, **kwargs: Any
-    ) -> ProjectFile:
+    def head(self, file_path: str, ref: str, **kwargs: Any) -> ProjectFile:
```

Comment:
While we refactor this, maybe just another nit: let's maybe put `head` after `get`, will also keep the diff reasonable here?